### ISSUE TITLE
[REVIEW]: RNAsik: A Pipeline for complete and reproducible RNA-seq analysis that runs anywhere with speed and ease

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#583]
**Submitting author:** @serine (<a href="http://orcid.org/0000-0002-9195-2626">Kirill Tsyganov</a>)
**Repository:** <a href="https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe" target ="_blank">https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe</a>
**Version:** 1.4.9
**Editor:** @pjotrp
**Reviewer:** @andrewyatz
**Archive:** <a href="https://doi.org/10.5281/zenodo.1403976" target="_blank">10.5281/zenodo.1403976</a>

## Status

[![status](http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce/status.svg)](http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce)

Status badge code:

```
HTML: <a href="http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce"><img src="http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce/status.svg"></a>
Markdown: [![status](http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce/status.svg)](http://joss.theoj.org/papers/32d910cf7f5acf29c9b49b82dfb647ce)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@andrewyatz, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.theoj.org/about#reviewer_guidelines. Any questions/concerns please let @pjotrp know.

  ### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Version:** Does the release version given match the GitHub release (1.4.9)?
- [x] **Authorship:** Has the submitting author (@serine) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks. @andrewyatz it looks like you're currently assigned as the reviewer for this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00583/joss.00583/10.21105.joss.00583.pdf)

**Comment by pjotrp** [comment]
Hi @serine, thanks for your interesting submission. @andrewyatz will review. To expedite the process, can you go through above checklist and make sure we ought to be able to tick the boxes? And ping us here when you are done. Thanks!

**Comment by andrewyatz** [comment]
Hi @serine I've been running your installation procedure and have two issues with it. Could you double check your installation scripts on a clean system and just make sure everything is in order please? I'd rather not be opening lots of issues on your repos if I can avoid it plus it lengthens the review process incredibly.

**Comment by pjotrp** [comment]
Ping @serine. The reviewer needs to be able to run the software. Can you provide the documentation to do just that? Should be straightforward with JVM. If you have already done that and can tick above boxes (e.g. for contribution guidelines) spell it out for the readers. Thanks!

**Comment by serine** [comment]
@pjotrp and @andrewyatz as I've mention in my [previous comment](https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe/issues/16#issuecomment-367302865) `RNAsik` depends on a lot of different tools and I some what assumed that dependencies of every tool don't need to be listed under `RNAsik`. In the case of BigDataScripit it depends on java, golang and ant, given that I said BigDataScript is required I assumed it dependencies. 

I since updated the docs to reflect all major/crucial dependencies in the docs, that of course requires a few system installs. I've tested the docs out and it worked for me. If you follow [quick start](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/docs/#installation) `RNAsik` should install. As I also mentioned in the docs now, I've only really tested this on ubuntu 16.04 (also works on 14.04). I don't have a mac so can't really test on that, but you wouldn't want to run it on mac anyway given the resources, unless the data is small of course e.g bacteria.  

so like I said docs have been updated with new install instruction. 

Let me know how it goes. also if you gonna meet some issues on mac do let me know so that I can accommodate mac users too. 

thanks

**Comment by serine** [comment]
Just to follow up on previous comment above. `virtualenv` isn't strictly necessary if you already have `ansible` installed, you can skip `virtualenv` step. Every time you opening a new shell you'll need to `export` both bds and `RNAsik` again. You can put those two export lines into your `~/.bashrc` if you like. I personally don't like to pollute my `~/.bashrc` with random exports. 

A general comment on `bio-ansible`, which I think I already made somewhere, you can change `bio.yml` to `all.ym` and add `--ask-become-pass` which will prompt you to your `sudo` password. This will do bunch of `apt-get` installs (again, ubuntu specific install) following by tools installation. The problem with doing `all.yml` this will install potentially many more `apt-get` packages than you really need for `RNAsik`

**Comment by pjotrp** [comment]
Complicated tools may have complicated setups ;). One solution would be to provide a Docker container (though I am no fan particularly).  @andrewyatz are up up for trying? Possibly a VM would be the easiest.

**Comment by serine** [comment]
yeh, I'm not sold on docker yet, but I can try getting you a docker image if that's preferred. Let me know @andrewyatz

**Comment by andrewyatz** [comment]
I'm going to try normally. You have my upmost sympathy with respect to dependency management and bioinformatic programs.

**Comment by pjotrp** [comment]
You may want to look into GNU Guix some day. I don't have any problems with the dependency graph.

**Comment by pjotrp** [comment]
@andrewyatz how are you doing?

**Comment by pjotrp** [comment]
Hi @serine. Looks like we have an impasse. Software deployment is the achilles heel of bioinformatics! 

Can you look into a deployment scheme? To me Docker, bioconda or GNU Guix are the systems of choice. Apt should not be used because it interferes with other software on a system. It should be easy to run software - at least a version of it. One advantage of choosing one of the three that it includes dependencies and it that gets promoted for future pipelines too. There is a paper coming out soon on comparing work flows, including conda and Guix. If you are interested I can send you a pre-press.

**Comment by serine** [comment]
@pjotrp It is a shame that you guys having issues with running pipeline, but it is great to get this kind of feedback about "ease of use" so to speak. Like I said before I can't imagine any other pipeline will be any easier to install. At the end of the day the pipeline doesn't require any installation at all, just BigDataScript, that can be downloaded as pre-compiled binary. But of course the pipeline of no use if other tools aren't there. 

I think docker beta ready. I'd have to dig into bio-ansible a bit (docker image is also build with bio-ansible, but you won't have to do it. I'll make it) I think internally we are would like to head bioconda way, in fact similar to docker, we are some what in alpha, but at the moment there is no conda channel for Golang and so there isn't a clean way of specifying dependency for BigDataScript at the moment. 

I did have a look at Guix briefly, I wasn't convinced.. I get a little confused when package manager gets mixed in with distro.. similar to https://nixos.org/nix/ which is both a package manager and a distro..

Yes please I'd like the pre-print if this is ok. Need to be convinced that docker or bioconda is the way to go.. I personally like to compile from source hence bio-ansible, but do understand "ease of use" and quick access is more important 

I have a few things piled up and am helping at running a python course next week. Also down here (Melbourne) we having Easter break from the 30th of March, so might be out of touch for a couple of week, but will try to get this all sorted and report back as soon as I can

Cheers

**Comment by andrewyatz** [comment]
Hi. Sorry this is more my fault that it's just not been possible for me to devote enough time to test through the existing changes that @serine has implemented. Don't take this as a comment on the changes but my availability.

**Comment by andrewyatz** [comment]
Okay I've resurrected my work and switched to an Ubuntu box. Still no joy. In fact the same error I see on my mac. This is in Mon

**Comment by andrewyatz** [comment]
Sorry can I prod again on this. There's little to no chance of me installing this. As far as I am aware there's been no response to the issues I've raised. The error isn't limited to just my mac as otherwise my VM attempts would have worked. Not sure where else to push this now

**Comment by serine** [comment]
@andrewyatz sorry about that, I've gone quite, my fault. I just responded to your [other issue](https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe/issues/22) could you have one more try. Thanks

**Comment by serine** [comment]
@andrewyatz It does appear that bio-ansible way to install `RNAsik` is a bit too much. Taking advice from @pjotrp and others around me I've created conda package, for linux only at this stage, which should simplify install great deal. 

Could you have a look at [updated docs](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/docs/#installation)

Basically you'll need to install miniconda and then RNAsik from my "channel" (conda terminology here)

```
bash Miniconda3-latest-Linux-x86_64.sh 
conda install -c serine rnasik 
conda install -c bioconda qualimap 
``` 

`QualiMap` is separate install, because it itself has fair few dependencies and it was tricky to include into `RNAsik` package

On my laptop it took about 35 minutes to install, this is subject to reasonably good internet connection as installation creates fair bit of traffic, downloading all of the different dependencies. 

Hopefully this will simplify things for you and others

Cheers

**Comment by serine** [comment]
by the way this is my channel page https://anaconda.org/serine
and this is rnasik package page https://anaconda.org/serine/rnasik

**Comment by serine** [comment]
@andrewyatz sorry, I think you also need to added a couple of channel

```
conda config --add channels defaults
conda config --add channels conda-forge
conda config --add channels bioconda
```

before installing from `conda` 

also updated docs. 

p.s default location of the config is `~/.condarc`

mine looks like this 

```
channels:
  - bioconda
  - conda-forge
  - defaults
```

**Comment by pjotrp** [comment]
@andrewyatz do you have time to look at this? We can look for another reviewer if necessary.

**Comment by serine** [comment]
@pjotrp @andrewyatz Let me guys know if you need anything else from me. 

Just to reiterate that best way to install is to use `conda install`. I actually shifted myself and now use `conda install` all the time, cause I tend to jump between diff machines. 

Also just to let you know that `RNAsik` have moved since I submitted for review. I'm now up to `1.5.1` version and making `1.5.2` soonish.. there have been some changes, but all in backward compatible manner. 

You can only get `1.5.1` through conda install, but that shouldn't effect the review right? 

Cheers

**Comment by andrewyatz** [comment]
Hi @pjotrp as the delay in my response suggests that I am no longer a good person to do this review. I am really sorry about this and apologise to Kirill and yourself about this. I appreciate the effort you have been spending on improving the process and feel conda is an excellent way to move in

**Comment by serine** [comment]
@andrewyatz no problem, thanks for all the feedback that you've provided so far. 

@pjotrp hopefully someone else can help out?

Cheers

**Comment by pjotrp** [comment]
Alright, let's look for someone who can run this pipeline, so we can wrap up review.

**Comment by pjotrp** [comment]
Hi @serine, sorry about the delay. So, you are saying that a conda install should work. Do you happen to have some example fastq files, i.e, something I can try? I am proposing I run it myself with your help. If it works I can switch to a reviewer role and I'll ask someone else to be the editor. It has happened before.

**Comment by serine** [comment]
@pjotrp yes, conda installable. I just wrote [quick_install.bash](https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe/blob/master/scripts/quick_install.bash) script that obviously does quick install, but I'm also using it with my travisCI testing. 

As you can see from [quick start section](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/docs/#quick-start) in the docs that you can just run the script and export executables into your PATH. The details of the script are explained under [installation section](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/docs/#installation) or just `cat` the script obviously 

The actual test data again explained in [data set for testing section](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/docs/#data-set-for-testing) and I've updated the README with the command to run, but here it is just in case

```
RNAsik -fqDir http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/fqFiles.txt \
       -align star \
       -fastaRef ftp://ftp.ensembl.org/pub/release-91/fasta/saccharomyces_cerevisiae/dna/Saccharomyces_cerevisiae.R64-1-1.dna_sm.toplevel.fa.gz \
       -gtfFile ftp://ftp.ensembl.org/pub/release-91/gtf/saccharomyces_cerevisiae/Saccharomyces_cerevisiae.R64-1-1.91.gtf.gz \
       -counts \
       -paired \
       -all
```

There are a couple of things I should mention. 

- BigDataScript (bds) just moved into version 2 (yay) but I had to patch up my pipeline a bit, because of that I'm using specific commit of bds that works. Right now it is hard coded into [quick_install.bash](https://raw.githubusercontent.com/MonashBioinformaticsPlatform/RNAsik-pipe/master/scripts/quick_install.bash) script. Also note that `RNAsik` is under active development and version 1.5.1 has moved on fair bit with bug fixes and new feature. I once again hard coded RNAsik specific commit into the install script. 

I'm waiting on bds to make next `v2.0rc9` release before making my next release `1.5.2` 

I understand that there are a lot ifs here. But things are out of my hands with respect to bds development. 

Basically if you just run the install script and test command I posted above it will work. Just note that STAR aligner needs fair bit of RAM, the test data set is small and I think 10-15 Gb of RAM should be plenty for mouse or human you really need ~ 32 Gb of RAM

thanks

**Comment by serine** [comment]
Forgot to mention new feature in `RNAsik` is ```-fqDir http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/fqFiles.txt```

where `-fqDir` can either take a list with files or directory. If text file is given, files expected to be one per line and if directory is give it will get traversed recursively looking for fastq files

the content of `fqFiles.txt` is 

```
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/ctrl1_R1.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/ctrl1_R2.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/ctrl2_R1.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/ctrl2_R2.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/Ind-Z1_R1.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/Ind-Z1_R2.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/Ind-Z2_R1.fastq.gz
http://bioinformatics.erc.monash.edu/home/kirill/sikTestData/rawData/IndustrialAntifoamAgentsYeastRNAseqData/Ind-Z2_R2.fastq.gz
```

`RNAsik` will handle the download for you. Also note that `fqFiles.txt` is on url as well, again download and parsing are all handled internally, just run the command as is above.

**Comment by pjotrp** [comment]
I'll give it a go this week.

**Comment by pjotrp** [comment]
Sorry about the delay. This is on my list now.

**Comment by serine** [comment]
great, thanks. 1.5.2 is out now by the way

**Comment by pjotrp** [comment]
I installed the software with bioconda bootstrapping from the GNU Guix conda package. Nice work! Mostly worked out of the box. I'll show a writeup when I am done.

@serine - do you have a test dataset I can just download and run? Sorry for the pain, but I don't want to be hunting down NCBI resources. Even a small set will do.

**Comment by pjotrp** [comment]
My report: I can report the software installs and runs to satisfaction. In fact, kudos @serine and colleagues for creating such an advanced pipeline which can be run on almost any Linux system out of the box with nice reporting output. I hope people will start using it. Since it is bionconda based you should also look at making it part of the common workflow initiative https://www.commonwl.org/ with @mr-c. 

Let me tick the relevant boxes which means we can process the publication for acceptance. @serine, sorry about the delays, it is not the usual, but then your software is not the usual either :). Good stuff.

**Comment by pjotrp** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00583/joss.00583/10.21105.joss.00583.pdf)

**Comment by pjotrp** [comment]
The review process is now complete. To finalize your submission and accept your paper in JOSS, we need two things. First, can you confirm that all references in your bibliography have a DOI (if one exists). 

Second, we need you to deposit a copy of your software repository (including any revisions made during the JOSS review process) with a data-archiving service. To do so:

 1. Create [a GitHub release](https://help.github.com/articles/creating-releases/) of the current version of your software repository
 2. Deposit that release with [Zenodo](https://zenodo.org/), [figshare](https://figshare.com/), or a similar DOI issuer.
 3. Post a comment here with the DOI for the release.

**Comment by serine** [comment]
This is great, thanks heaps !

About the second part; I've done Zenodo here https://zenodo.org/record/1315013#.W0_PY3VuaV4

[![DOI](https://zenodo.org/badge/DOI/10.21105/joss.00583.svg)](https://doi.org/10.21105/joss.00583)

I was a bit confused about which DOI to use.. I just tool whatever was there in the pds that you've generated.

About the first bit _can you confirm that all references in your bibliography have a_ 

Looking through the references these tools don't have doi

```
Broadinstitute. n.d. “Picard-Tools.” broadinstitute. http://broadinstitute.github.io/
picard.
```
```
Li, Heng. 2013. “Aligning Sequence Reads, Clone Sequences and Assembly Contigs with
BWA-MEM,” March. http://arxiv.org/abs/1303.3997.
```
```
Powell, David. 2015. “Degust: Powerfull and User Friendly Front-End Data Analsysis,
Visualisation and Exploratory Tool for Rna-Sequencing.” github. http://degust.erc.
monash.edu
```

do I need to do something about this?

**Comment by mr-c** [comment]
Would an RRID URI be acceptable? I like to cite both the idea of a piece of software and any relevant methods papers.

Picard:

https://identifiers.org/rrid/RRID:SCR_006525

Degust:

https://identifiers.org/rrid/RRID:SCR_001878


For bwa-mem I suggest listing  both the Arxiv paper and the RRID
https://arxiv.org/abs/1303.3997
https://identifiers.org/rrid/RRID:SCR_010910

(There are citation instructions for BWA but it doesn't mention the `mem` algorithm: http://bio-bwa.sourceforge.net/ just `short` and  `sw`)

**Comment by serine** [comment]
great thanks a lot guys, and yes shout out to everyone involved, every little bit of advise, comment and gitissue really helped in shaping this into a better tool !

**Comment by serine** [comment]
@mr-c was that comment _Would an RRID URI be acceptable?_ to me or at @arfon cause I'm happy with what ever works

**Comment by serine** [comment]
@mr-c how about now https://zenodo.org/record/1315013#.W1A5FHVuaV4 ? 

using 

```
pandoc -o check.pdf --bibliography=paper.bib paper.md
```
builds to this

[check.pdf](https://github.com/openjournals/joss-reviews/files/2208689/check.pdf)

not sure how it'll look with JOSS compilation.. also don't think I can do much about footer overlap

if you think this looks ok I can commit this change to master and perhaps ask @pjotrp to rebuild final pdf if that's ok?

**Comment by mr-c** [comment]
@serine much more readable, thank you.

**Comment by serine** [comment]
d567e44389a761ceaefbe31d9c3e380ea69a9645

**Comment by pjotrp** [comment]
I think you can do it with 

       @whedon generate pdf

**Comment by serine** [comment]
@whedon generate pdf

**Comment by serine** [comment]
@whedon generate pdf pretty please ?

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00583/joss.00583/10.21105.joss.00583.pdf)

**Comment by serine** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00583/joss.00583/10.21105.joss.00583.pdf)

**Comment by serine** [comment]
haha always works if you ask nicely

**Comment by pjotrp** [comment]
@arfon I think we are ready to R&R

**Comment by arfon** [comment]
@serine - you appear to have associated the JOSS DOI with your archive in Zenodo (https://zenodo.org/record/1315013#.W2m3Nv5Khjt). Could you please follow [this guide](https://guides.github.com/activities/citable-code/) and create an archive of the GitHub repository (https://github.com/MonashBioinformaticsPlatform/RNAsik-pipe) that has a Zenodo DOI and not a JOSS one?

**Comment by serine** [comment]
right, thanks for the link to the guide @arfon . I've done everything but the new release. I'll have to do a bit a work for my next release. I can't just slap new release tag there is a [timeline and things](https://monashbioinformaticsplatform.github.io/RNAsik-pipe/roadmap/) :) 
Will try to get it done in the next couple of days. 

Pretty exciting to be so close !

Thanks

**Comment by serine** [comment]
@pjotrp sorry about the wait, was away for a week and was playing catch up. I just made another release that is hooked to zenodo. https://zenodo.org/account/settings/github/repository/MonashBioinformaticsPlatform/RNAsik-pipe

I didn't realise before that I could have "annotated" any of my previous releases and that would have showed up in zenodo. I thought that after setting up a webhook zenodo was "waiting" for an event (next release). Anyway I think all done now.

@arfon is there anything else I need to do?

Thanks

**Comment by pjotrp** [comment]
@arfon I believe we are good to accept this submission!

**Comment by pjotrp** [comment]
@whedon set 10.5281/zenodo.1403976 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.1403976" target="_blank">10.5281/zenodo.1403976</a> is the archive.

**Comment by serine** [comment]
Awesome !!! thanks heaps everyone !

**Comment by arfon** [comment]
@andrewyatz @mr-c - many thanks for your reviews here and to @pjotrp for editing this submission ✨ 

@serine - your paper is now accepted into JOSS and your DOI is https://doi.org/10.21105/joss.00583 :zap: :rocket: :boom:

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](http://joss.theoj.org/papers/10.21105/joss.00583/status.svg)](https://doi.org/10.21105/joss.00583)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.00583">
  <img src="http://joss.theoj.org/papers/10.21105/joss.00583/status.svg" alt="DOI badge" >
</a>
```
This is how it will look in your documentation:

[![DOI](http://joss.theoj.org/papers/10.21105/joss.00583/status.svg)](https://doi.org/10.21105/joss.00583)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: http://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://www.flipcause.com/secure/cause_pdetails/Mjk3Nzk=
